NM trade enhancement.user.js

```diff
@@ -1594,13 +1594,26 @@ async function addTradePreview (notification) {
     singleton.setInstances(Object.values(tips));
 }
 
+/**
+ * Allows you to hit enter with your keyboard to dismiss confirm boxes.
+ * @param {Event} e - keydown event
+ */
+async function okayNotification (e) {
+    const ENTER_KEYCODE = 13;
+    if (e.keyCode === ENTER_KEYCODE) {
```

Comment:
`.keyCode` is marked as deprecated though it likely will be available for years for backward compatibility.
I personally would use `e.code === "Enter"`.